CHANGELOG.rst

```diff
@@ -2,6 +2,23 @@
 Release Notes
 =============
 
+6.12.2
+======
+
+Prelude
+-------
+
+Release on: 2019-07-03
+
+This release is only available on Windows.
```

Comment:
it's obvious to us, but since 6.12.0 and 6.12.1 were not released on Windows, I would add a mention here that `this release also includes all the changes shipped in v6.12.0 and v6.12.1`, or something of that nature